### ISSUE TITLE
[IMP] AFC-1476b (AFC-1544) -  user-wizard second round

### ISSUE THREAD

**Issue by FredrikArvas** [vertelab/odoo-af#470]
1 **Userstory:** 
As a user, I want to assign user rights in the normal form-view instead of with the action-button (because action-buttons are difficult to use with a screen-reader)
This is based on https://github.com/vertelab/odoo-af/issues/456

1 Replace the Action-function (and its form) with a section on the normal Form with these roles. 

2 Change the ”ServerAction”: When the form is saved, do these steps: 
A check that the logged in user belongs to the SuperAdmin group
B check that the roles saved are part of the original ”7 DAFA groups”
C Check that there is an email-address
D If there is no res.user, create one
F Update groups (add/remove).

3 Add description-text to the roles (use "the group name" + "description" as dummy-text) and display that in the form.
https://github.com/vertelab/odoo-base/tree/Dev-12.0-Fenix-Sprint-02/base_user_groups_dafa

![image](https://user-images.githubusercontent.com/9428370/101987461-83438a00-3c94-11eb-88b3-4a28afc93e00.png)

3 **Bugreport:** (only for Bug-Issues)
We still get this error on our side: (I have tried 4 different servers)
![image](https://user-images.githubusercontent.com/9428370/101987037-f7306300-3c91-11eb-9ac9-96410d861c9d.png)

**Module name:** (added by Nils or Fredrik)
odoo-auth/auth_user_rights_wizard

**Development Branch name:** 
Dev-12.0-DAFA-Sprint-02-AFC-1476b-user-right-rewrite

**Target Branch name:** 
https://github.com/vertelab/odoo-auth/tree/Dev-12.0

**Test-steps:**
Install the module
Login with admin
Create a **user**, assign DAFA SuperAdmin-role (no access to settings)
Login with this user 
Create a **Employee**, add DAFA Participant Write-role.
Result = a user should be linked to this employee, with the right user-role. 
No dialoge boxes.

**Description to the manifest:** 
This version replaces the Action-popup for checkboxes in the form.

**Estimate:** (@developers: Add your times here, not in comments please)
Estimated Hours: 
Estimated Date: 
Actual Hours: 2hrs

**Comment by iammiracle** [comment]
@FredrikArvas [PR created](https://github.com/vertelab/odoo-auth/pull/8). The description for the groups has been added to the module.. 

The error you were getting I see it when the email fields is empty, but once the email field has a value, then it will save without errors. Moreover the validation for email is in place. Kindly test

**Comment by iammiracle** [comment]
Hi @FredrikArvas I have added tour to this user wizard module. I updated the PR. Kindly test

**Comment by FredrikArvas** [comment]
Hi @iammiracle 
This will be interesting. 
Please provide the usual Screenshots, PR and instructions on how to use it.

I would like to have the tour in a separate module, since we are not sure how/when to use it yet.

Here is a GitIssue for that https://github.com/vertelab/odoo-af/issues/483

**Comment by FredrikArvas** [comment]
@iammiracle 
We have tested this module on several servers and we still get the error. 
Daniel tried to solve it by adding more "sudo"-commands, but it only changed the error message to the current version "You are not permitted to do this". 

1 Could you pleace check this module again?
https://github.com/vertelab/odoo-auth/tree/Dev-12.0-Fenix-Sprint-02/auth_user_rights_wizard_tour
The requriements: 
The logged in User should not have "user-admin"-rights. 
The logged in User should be assigned "DAFA SuperAdmin"-rights
![image](https://user-images.githubusercontent.com/9428370/103166185-db939380-481f-11eb-82d4-966513986d85.png)

2 
Could you see if you can change the text-field into check-boxes and provide information about the different roles that are available? (See point 2 in the description above).

**Comment by iammiracle** [comment]
@FredrikArvas You are getting that because the logged in user does not have the right to do ao. Give the user the DAFA Super Admin and you will be able to assign rights.

You are referring to the tour module when you are supposed to be auth_user_rights_wizard, the tour module only depends on the auth_user_rights_wizard module. 

I'm not sure I understand the second point. Which text field are you referring to.

**Comment by FredrikArvas** [comment]
![image](https://user-images.githubusercontent.com/9428370/103178471-53df7080-4883-11eb-8345-46e5d8dee075.png)

Hi Miracle
Sorry for confusing the modules (holidaymode).

I have assigned the SuperAdmin-role to the DAFA-test-user. 
Please check again on a clean database. I know you did not have the problem, but we have on 5 different databases.
:-)

**Comment by iammiracle** [comment]
Hi @FredrikArvas I have checked on a new db, see video [here](https://www.loom.com/share/365d77c579944934acc8a143b561488a)
